recipe/meta.yaml

```diff
@@ -7,20 +8,21 @@ package:
 source:
   fn: traitlets-{{ version }}.tar.gz
   url: https://pypi.io/packages/source/t/traitlets/traitlets-{{ version }}.tar.gz
-  md5: d0725f9818bd9e112a6531da26d433b8
+  sha256: {{ sha256 }}
 
 build:
   number: 0
-  script: pip install --no-deps .
+  script: python setup.py install
```

Comment:
@SylvainCorlay note that we want to go in the `pip install --no-deps .` direction everywhere. We are just waiting for some fixes in `conda-build` to do so. There are many advantages and, IMO as a packager, the best one is to get the `entry_points` for free.